### DISCUSSION TITLE
Alerting when two windows events happen within a certain timeframe

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#6315] [Unsupported Versions] [answered]
Hello, 

I have an Remote Desktop server where users are dropping and immediately reconnecting.  The server is logging event ID 24 for the disconnect event and immediately after, logging event ID 25 for the reconnect.  I believe that I've found the root cause (AV).  

My question here: Is there a way to trigger an alert when two such events happen within a certain timeframe?  I welcome a point in the right direction.

Thank you, ...Rob

**Comment by defensivedepth** [comment, marked as answer]
You could try creating an Elastalert rule for this. 

eg chaining multiple rules together using the metadata (checking to see a past rule has fired)
